### PR TITLE
test: add test book and patron validators

### PR DESCRIPTION
**Pull request by ronaldo-mendoza-jalafund**
Adding tests for book and pattern validations

### CODE REVIEW

**Review by alex-paca-jalafund — CHANGES_REQUESTED** [review]
Update with lastest changes on release branch

**Comment by joseluis-teran-jalafund** [comment]
@ronaldo-mendoza-jalafund Please review the conventional commits guide to make a commit.

**Comment by joseluis-teran-jalafund** [comment]
You can review this **[page](https://dev.to/achamorro_dev/conventional-commits-que-es-y-por-que-deberias-empezar-a-utilizarlo-23an)** to pick one according the context

**Review by joseluis-teran-jalafund — APPROVED** [review]
Nice!

File: LosGosusTest/src/Validator/BookValidatorTest.cs [hunk 1]
```diff
@@ -0,0 +1,110 @@
+using LosGosus.Models;
+using LosGosus.Validators.Concretes;
+
+public class BookValidatorTests
```
Comment (alex-paca-jalafund) [inline LosGosusTest/src/Validator/BookValidatorTest.cs]: There is already a BookValidatorTests class on /services/BookValidatorTests.cs

**Review by alex-paca-jalafund — CHANGES_REQUESTED** [review]

File: LosGosusTest/src/Validator/BookValidatorTest.cs (hunk 1 above)
Comment (joseluis-teran-jalafund) [reply LosGosusTest/src/Validator/BookValidatorTest.cs]: Guess he should delete it. That class was with a previous version of the refactor.

**Review by alex-paca-jalafund — APPROVED** [review]
Nice, it works.

**Pull request merged**